acstools/findsat_mrt.py

```diff
@@ -1107,73 +1143,65 @@ def _remove_angles(self, ignore_theta_range=None):
 
         self.source_list = self.source_list[~remove]
 
-    def run_all(self, **kwargs):
+    def run_all(self, trim_catalog=False, plot=False, plot_streak=False, close_plot=True):
         '''
         Run the entire pipeline to identify, filter, and mask trails.
+        This calls the following methods in the given order:
 
-        Parameters
-        ----------
-        **kwargs : dict, optional
-            Additional keyword arguments run_mrt, find_mrt_sources,
-            filter_sources, mask_mask, and save_output
+        1. :meth:`run_mrt`
+        2. :meth:`find_mrt_sources`
+        3. :meth:`filter_sources`
+        4. :meth:`make_mask`
+        5. :meth:`save_output`
 
-        Returns
-        -------
-        None.
+        See the documentation for methods above on how to use the
+        keyword options.
 
         '''
-
-        self.run_mrt(**kwargs)
-        self.find_mrt_sources(**kwargs)
-        self.filter_sources(**kwargs)
-        self.make_mask(**kwargs)
-        self.save_output(**kwargs)
+        self.run_mrt(plot=plot)
+        self.find_mrt_sources(plot=plot)
+        self.filter_sources(trim_catalog=trim_catalog, plot=plot, plot_streak=plot_streak)
+        self.make_mask(plot=plot)
+        self.save_output(close_plot=close_plot)
 
 
 class WfcWrapper(TrailFinder):
-
     '''
-    Wrapper for trail_finder designed specifically for ACS/WFC data.
+    Wrapper for `TrailFinder` designed specifically for ACS/WFC data.
 
     This class enables quick reading and preprocessing of standard
     full-frame ACS/WFC images.
 
     .. note::
 
-        * This class is not designed for use with subarray images.
+        This class is not designed for use with subarray images.
 
     Parameters
     ----------
-    image_file : string
-        ACS/WFC data file to read. Should be a fits file.
+    image_file : str
+        ACS/WFC data file to read. Should be a FITS file.
     extension : int, optional
```

Comment:
We should add that extension is actually a required keyword for FLC/FLT data (not for DRC/DRZ)